src/Controller/HelpController.php

```diff
@@ -20,10 +22,11 @@ class HelpController extends AbstractController
 {
     private $session;
 
-    public function __construct(SessionInterface $session)
+    public function __construct(RequestStack $requestStack)
```

Comment:
Eigenlijk zou dit helemaal niet in de constructor moeten maar gewoon in de controller methodes zelf. Dan heb je helemaal geen private variabele nodig. Maar dat is misschien een iets te grote refactor